packages/babel-cli/src/babel/options.js

```diff
@@ -124,6 +125,7 @@ commander.option(
 commander.option(
   "-d, --out-dir [out]",
   "Compile an input directory of modules into an output directory",
+  getOutDir(),
```

Comment:
This is going to break the check at https://github.com/iansu/babel/blob/3ec44323aa3477de09dc5b836a0f7233fc6b2967/packages/babel-cli/src/babel/index.js#L9 because there will always be a dir. We should invert that so that if the user specifies `--out-file`, it uses the file approach.

This is going to break Babel's ability to write to stdout: https://github.com/babel/babel/blob/6b91d6434d2682ef6b0bf4ff5c9540b8ddbcd7f9/packages/babel-cli/src/babel/file.js#L90 because there will always be a file. We'll want to allow `--out-file -` to say "output to stdout".